src/widget/image-map/image-map.js

```diff
@@ -254,6 +257,19 @@ class ImageMap extends Widget {
         } );
     }
 
+    /**
+     * If somehow the image not fully loaded before applying print style
+     * Then we need to recalculate the height
+     */
+    _recalculateHeight() {
+        const questionHeight = this.question.offsetHeight;
+        const fieldsetHeight = this.question.querySelector( 'fieldset' ).offsetHeight;
+        if ( this.question.classList.contains( 'print-height-adjusted' ) && ( fieldsetHeight > questionHeight ) ) {
+            // 19 px got from expected height - currentOffsetHeight
+            this.question.style.height = `${fieldsetHeight + 19}px`;
```

Comment:
I don't quite understand the 19px (after reading comment). Can you get this value programmatically?